src/app/WriteClient.cpp

```diff
@@ -350,7 +359,9 @@ CHIP_ERROR WriteClientHandle::SendWriteRequest(SessionHandle session, System::Cl
 {
     CHIP_ERROR err = mpWriteClient->SendWriteRequest(session, timeout);
```

Comment:
Within `SendWriteRequest`, we should be validating that the set of paths written are _all_ groupcast paths (i.e they contain no EndpointId in the path).

Reply:
Yes, an issue was created to address this #11604